### PR TITLE
Adds 2 new brain trauma lines

### PR DESCRIPTION
**Pull request by TheVekter**
## About The Pull Request
A couple of new classics from the admin forums.

## Why It's Good For The Game
The first one gave me brain rot and I can't stop saying it, the second one made me make a long noise so it goes in too

## Changelog
:cl: Vekter
add: Added 2 new brain damage lines. Beat your skull in with a wrench and discover them today!
/:cl:

### CODE REVIEW

File: strings/traumas.json [hunk 1]
```diff
@@ -65,7 +65,9 @@
         "WHERES THE SLIP REWRITE WHERE THR FUCK ID IT?",
         "who the HELL do u thenk u r?!!!!",
         "without oxigen blob don't evoluate?",
-        "youed call her a toeugh bithc"
+        "youed call her a toeugh bithc",
+		"built DIFFERENT. I Stack Paper!",
+		"MAKE JIANT CRISTAL BURN I KEPT THROWING WELDERS IT WOOD LIGHT"
```
Comment (dragomagol) [inline strings/traumas.json:70]:
```suggestion
        "youed call her a toeugh bithc",
	"built DIFFERENT. I Stack Paper!",
	"MAKE JIANT CRISTAL BURN I KEPT THROWING WELDERS IT WOOD LIGHT",
```

Comment (TheVekter) [reply strings/traumas.json:70]: image
???????

Comment (TheVekter) [reply strings/traumas.json]: I fixed it but what the fuck

File: strings/traumas.json [hunk 2]
```diff
@@ -65,7 +65,9 @@
         "WHERES THE SLIP REWRITE WHERE THR FUCK ID IT?",
         "who the HELL do u thenk u r?!!!!",
         "without oxigen blob don't evoluate?",
-        "youed call her a toeugh bithc"
+        "youed call her a toeugh bithc",
+	"built DIFFERENT. I Stack Paper!",
+	"MAKE JIANT CRISTAL BURN I KEPT THROWING WELDERS IT WOOD LIGHT"
```
Comment (Jacquerel) [inline strings/traumas.json:70]:
```suggestion
	"MAKE JIANT CRISTAL BURN I KEPT THROWING WELDERS IT WOOD LIGHT",
```
Still missing the trailing comma, just makes expanding it easier later

**Review by Jacquerel — CHANGES_REQUESTED** [review]

File: strings/traumas.json (hunk 2 above)
Comment (Jacquerel) [reply strings/traumas.json:70]: wait fuck its json i guess we dont do that there

Comment (TheVekter) [reply strings/traumas.json:70]: I already had this discussion with Tattle, it's a JSON list, not a BYOND one. Trailing comma errors out.

image

**Comment by Jezzared** [comment]
Looks like the brain trauma lines are in alphabetical order, so the ones you're adding are out of place since they're just stuck onto the end.

**Comment by TheVekter** [comment]
> Looks like the brain trauma lines are in alphabetical order, so the ones you're adding throw that off since they're just stuck onto the end.

Nice catch.

**Comment by TheVekter** [comment]
> i really want context for the first one. it has me fascinated to the morbid extent.

The "I Stack Paper!" one doesn't have any; it's just a non-sequitur from a Grobelon ahelp.

The "MAKE JIANT CRYSTAL" one comes from this ticket:

image

**Comment by DustanHache** [comment]
maybe i am built different then for thinking there was more to stacking paper. still, both of these are highly amusing.

**Pull request merged**